ui/components/MeshModelRegistry/MesheryTreeView.js

```diff
@@ -1,27 +1,63 @@
 import React, { useEffect, useState } from 'react';
 import { TreeView } from '@mui/x-tree-view/TreeView';
-import { Box, Typography, IconButton, FormControlLabel, Switch } from '@material-ui/core';
+import {
+  Box,
+  Typography,
+  IconButton,
+  FormControlLabel,
+  Switch,
+  SvgIcon,
+  useTheme,
+} from '@material-ui/core';
 import Checkbox from '@mui/material/Checkbox';
 import { MODELS, COMPONENTS, RELATIONSHIPS, REGISTRANTS } from '../../constants/navigator';
 import SearchBar from '../../utils/custom-search';
-import ExpandAllIcon from '../../assets/icons/expand_all';
-import CollapseAllIcon from '../../assets/icons/collapse_all';
-import ExpandMoreIcon from '../../assets/icons/expand_more';
-import ChevronRightIcon from '../../assets/icons/chevron_right';
 import { StyledTreeItemRoot } from './MeshModel.style';
-import useStyles from '../../assets/styles/general/tool.styles';
 
+function MinusSquare(props) {
+  return (
+    <SvgIcon fontSize="inherit" style={{ width: 14, height: 14 }} {...props}>
+      {/* tslint:disable-next-line: max-line-length */}
+      <path d="M22.047 22.074v0 0-20.147 0h-20.12v0 20.147 0h20.12zM22.047 24h-20.12q-.803 0-1.365-.562t-.562-1.365v-20.147q0-.776.562-1.351t1.365-.575h20.147q.776 0 1.351.575t.575 1.351v20.147q0 .803-.575 1.365t-1.378.562v0zM17.873 11.023h-11.826q-.375 0-.669.281t-.294.682v0q0 .401.294 .682t.669.281h11.826q.375 0 .669-.281t.294-.682v0q0-.401-.294-.682t-.669-.281z" />
+    </SvgIcon>
+  );
+}
```

Comment:
@aabidsofi19 I think we should move these icons to assets unless there is reason to keep them here ^^

Reply:
yup , forgot about it